doc/datalog2.0-workshop/.gitignore

```diff
@@ -2,4 +2,4 @@
 *.blg
 *.log
 *.aux
-top.pdf
+paper.pdf
```

Comment:
This has been published already.

Reply:
It is mainly here to avoid showing the paper.pdf as modified every time the latex sources are built. So I thought, we can only commit paper.pdf if need be with `git add -f`. It's a little ugly, I agree. I can remove paper.pdf from gitignore, but we should be a bit cautious when rebuilding the latex sources (to not add paper.pdf without any diffs).